docs/Integration_with_GitHub.rst

```diff
@@ -130,7 +130,8 @@ EasyBuild will validate the provided token, to check that authenticated access t
 Specify location of working directories (``--git-working-dirs-path``)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can specify the location of your Git working directories using ``--git-working-dirs-path``.
+You can specify the location of your Git working directories using ``--git-working-dirs-path`` or
+you can set it with the ``git-working-dirs-path`` option in your EasyBuild configuraton file.
```

Comment:
There actually a 3rd option (just like for any other configuration setting): specifying the location via the `$EASYBUILD_GIT_WORKING_DIRS_PATH` environment variable, see http://easybuild.readthedocs.io/en/latest/Configuration.html#consistentency-across-supported-configuration-types...

So either we mention all three options, or maybe just refer to that section?

Rewording `using --git-working-dirs-path` to `using the git-working-dirs-path configuration setting` probably also helps?

Reply:
ok, so maybe it can be mentioned the same as for `Providing a GitHub username` ?
with three bullets items?

Reply:
Sure, that makes sense.

On the other hand, if we need to do this for every configuration setting we mention somewhere, that may be a bit too much...

Feel free to change it in this specific case those, I'll be happy to merge it!